PURL-TYPES.rst

```diff
@@ -516,9 +516,8 @@ pypi
 ``pypi`` for Python packages:
 
 - The default repository is ``https://pypi.org``. (Previously  ``https://pypi.python.org``.)
-- PyPI treats ``-`` and ``_`` as the same character and is not case sensitive.
-  Therefore a PyPI package ``name`` must be lowercased and underscore ``_``
```

Comment:
PURL's rule was a "must be",
the new one is a "should be".

depending on the POV, this would be a breaking change.

I'd suggest to stay with the old wording "must", just do be safe here.